test/integration_test.py

```diff
@@ -998,6 +998,26 @@ def assert_manifest(expected_bundles):
             # Without credentials, only the public bundle should be represented
             assert_manifest({public_bundle})
 
+            # Create a single-file curl manifest and verify that the OAuth2
+            # token is present in the .curlrc
+            managed_access_file_id = self.random.choice(
+                self.random.choice(hits)['files']
+            )['uuid']
+            manifest_url.set(args={
+                'filters': json.dumps({'fileId': {'is': [managed_access_file_id]}}),
+                'format': 'curl'
+            })
+            with self._service_account_credentials:
+                response = self._get_url_content(str(manifest_url))
+            self._check_curl_manifest(catalog, response)
+            auth_header = one(
+                line
+                for line in TextIOWrapper(BytesIO(response))
+                if line.startswith('--header')
+            ).split(' ', 1).pop().strip('"\n')
```

Comment:
```suggestion
            auth_header_line = one(
                line
                for line in TextIOWrapper(BytesIO(response))
                if line.startswith('--header')
            )
            auth_header = auth_header_line.split(' ', 1).pop().strip('"\n')
```

Reply:
I'd also be ok with inlining auth_header since the assertion is already taking up two lines.